website/templates/trademark_search.html

```diff
@@ -0,0 +1,109 @@
+{% extends "base.html" %}
+{% load i18n %}
```

Comment:
I'm not sure if this one is in use as well.

Reply:
I don't why I forget this but now removed it 👍